lib/datasource/npm.js

```diff
@@ -185,17 +188,22 @@ async function getDependencyInner(name, retries = 5) {
       homepage: res.homepage,
       latestVersion: res['dist-tags'].latest,
       repositoryUrl,
-      versions: res.versions,
-      'dist-tags': res['dist-tags'],
       'renovate-config': latestVersion['renovate-config'],
     };
-    Object.keys(dep.versions).forEach(version => {
-      const v = {};
+    const versions = Object.keys(res.versions)
+      .filter(version => isVersion(version))
```

Comment:
This is fine to leave in, but do you think npmjs ever returns a non-semver version?